packages/http-client-csharp/generator/TestProjects/CadlRanch/http/type/model/visibility/src/Generated/Models/VisibilityModel.cs

```diff
@@ -27,5 +27,7 @@ public bool DeleteProp
             get => throw null;
             set => throw null;
         }
+
+        public VisibilityModelNoneProp NoneProp => throw null;
```

Comment:
you might need to figure out why this show up again

Reply:
I didn't merge latest package.json before. Now this change doesn't exist.